packages/core/src/2d/text/Font.ts

```diff
@@ -11,25 +11,44 @@ export class Font extends RefObject {
   private static _fontMap: Record<string, Font> = {};
 
   /**
-   * Create a font from OS.
+   * Create a font.
    * @param engine - Engine to which the font belongs
-   * @param name - The name of font
+   * @param name - The name of font want to create
+   * @param fontUrl - The font url to register, if not, will use system font
    * @returns The font object has been create
    */
-  static createFromOS(engine: Engine, name: string = ""): Font {
+  static async create(engine: Engine, name: string, fontUrl: string = ""): Promise<Font> {
```

Comment:
OS font shouldn't async.

The API of Font resource create:
- const systomFont = Font.createFromOS();
- const ttfFont = Resource.load("xxx/yyy/font.ttf");
- const customFont = Resource.load("xxx/yyy/font.font");